crates/arrow_sql_gen/src/mysql.rs

```diff
@@ -254,6 +255,81 @@ pub fn rows_to_arrow(rows: &[Row]) -> Result<RecordBatch> {
                         i
                     );
                 }
+                ColumnType::MYSQL_TYPE_BLOB => {
+                    handle_primitive_type!(
+                        builder,
+                        ColumnType::MYSQL_TYPE_BLOB,
+                        BinaryBuilder,
+                        Vec<u8>,
+                        row,
+                        i
+                    );
+                }
+                ColumnType::MYSQL_TYPE_TIMESTAMP => {
```

Comment:
Something is wrong with time parsing

![CleanShot 2024-04-09 at 02 21 26@2x](https://github.com/spiceai/spiceai/assets/827338/eb9b432a-9129-4daa-aeb2-880c78bfd960)

Reply:
Fixed:
![CleanShot 2024-04-09 at 02 40 18@2x](https://github.com/spiceai/spiceai/assets/827338/8a6f9942-6370-49bc-b1ca-726dad45cbe9)